### PR TITLE
Add recipe for r-rsvd

### PR DESCRIPTION
**Pull request by npavlovikj**
Add recipe for r-rsvd

### CODE REVIEW

File: recipes/r-rsvd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+source:
+  fn: rsvd_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/rsvd_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/rsvd/rsvd_{{ version }}.tar.gz
+  sha256: c91f0abd50f5b8008a99b2f3bd046b1ca91dc0d0de9714b0d2fcbe2332f1e1b4
+
+build:
+  number: 0
+  skip: True  # [win]
```
Comment (bgruening) [inline recipes/r-rsvd/meta.yaml]: Can we try to build this on windows.

File: recipes/r-rsvd/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+test:
+  commands:
+    - $R -e "library('rsvd')"  # [not win]
+
+about:
+  home: https://github.com/erichson/rSVD
+  license: GPL (>= 2)
+  summary: Low-rank matrix decompositions are fundamental tools and widely used for data analysis, dimension reduction, and data compression.
+  license_family: GPL3
```
Comment (bgruening) [inline recipes/r-rsvd/meta.yaml]: We need to add license_file

**Comment by npavlovikj** [comment]
Thanks for the feedback @bgruening ! I can add the "win" build, and should I just use
  _license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'  # [unix]
  license_file: '{{ environ["PREFIX"] }}\R\share\licenses\GPL-3'  # [win]_
for the license_file section ?

**Comment by bgruening** [comment]
Yes, looks better now. Thanks!

**Pull request merged**